src/main/java/org/infernalstudios/foodeffects/FoodEffectsEvents.java

```diff
@@ -34,109 +31,85 @@
-                () -> (int)(COMMON.COOKIE_EFFECT_DURATION.get() * 20),
-                () -> COMMON.COOKIE_EFFECT_AMPLIFIER.get()
-            )
-        );
-        EFFECT_MAP.put(
-            Items.PUMPKIN_PIE,
-            new EffectData(
-                () -> MOB_EFFECTS.getValue(new ResourceLocation(COMMON.PUMPKIN_PIE_EFFECT.get())),
-                () -> (int)(COMMON.PUMPKIN_PIE_EFFECT_DURATION.get() * 20),
-                () -> COMMON.PUMPKIN_PIE_EFFECT_AMPLIFIER.get()
-            )
-        );
-        EFFECT_MAP.put(
-            Items.HONEY_BOTTLE,
-            new EffectData(
-                () -> MOB_EFFECTS.getValue(new ResourceLocation(COMMON.HONEY_BOTTLE_EFFECT.get())),
-                () -> (int)(COMMON.HONEY_BOTTLE_EFFECT_DURATION.get() * 20),
-                () -> COMMON.HONEY_BOTTLE_EFFECT_AMPLIFIER.get()
-            )
-        );
-        EFFECT_MAP.put(
-            Items.BAKED_POTATO,
-            new EffectData(
-                () -> MOB_EFFECTS.getValue(new ResourceLocation(COMMON.BAKED_POTATO_EFFECT.get())),
-                () -> (int)(COMMON.BAKED_POTATO_EFFECT_DURATION.get() * 20),
-                () -> COMMON.BAKED_POTATO_EFFECT_AMPLIFIER.get()
-            )
-        );
-    }
-
+    protected static final Map<Item, List<EffectData>> EFFECT_MAP = new ConcurrentHashMap<>();
 
     @SubscribeEvent
     public void onLivingEntityUseItemFinish(LivingEntityUseItemEvent.Finish event) {
         if (event.getEntity() instanceof Player player) {
-            ItemStack stack = event.getItem();
-    
-            if (stack.getItem() == Items.DRIED_KELP) {
-                player.removeEffect(MobEffects.POISON);
-                player.removeEffect(MobEffects.CONFUSION);
-                player.removeEffect(MobEffects.BLINDNESS);
+            onEat(player, event.getItem().getItem());
+        }
+    }
+
+    @SubscribeEvent
+    public void onFMLClientSetupEvent(FMLClientSetupEvent event) {
+        System.out.println("FMLClientSetupEvent");
+    }
+    @SubscribeEvent
+    public void onFMLCommonSetupEvent(FMLCommonSetupEvent event) {
+        System.out.println("FMLCommonSetupEvent");
+    }
+    @SubscribeEvent
+    public void onFMLConstructModEvent(FMLConstructModEvent event) {
+        System.out.println("FMLConstructModEvent");
+    }
+    @SubscribeEvent
+    public void onFMLDedicatedServerSetupEvent(FMLDedicatedServerSetupEvent event) {
+        System.out.println("FMLDedicatedServerSetupEvent");
+    }
+    @SubscribeEvent
+    public void onFMLLoadCompleteEvent(FMLLoadCompleteEvent event) {
+        System.out.println("FMLLoadCompleteEvent");
+    }
+    @SubscribeEvent
+    public void onInterModEnqueueEvent(InterModEnqueueEvent event) {
+        System.out.println("InterModEnqueueEvent");
+    }
+    @SubscribeEvent
+    public void onInterModProcessEvent(InterModProcessEvent event) {
+        System.out.println("InterModProcessEvent");
+    }
```

Comment:
Are these all necessary to keep?

Reply:
ah!! that was a debug thing that i forgot to remove